### PR TITLE
Password strength

### PR DESCRIPTION
**Pull request by fiskus**
Screenshot from 2022-02-15 18-24-04

### CODE REVIEW

File: catalog/app/containers/Auth/SignUp.js [hunk 1]
```diff
@@ -28,6 +29,48 @@ const Container = Layout.mkLayout('Complete sign-up')
...
+    <>
+      <Layout.Field
+        InputProps={{
+          endAdornment: isWeak && (
+            <M.InputAdornment position="end">
+              <WeakPasswordIcon />
+            </M.InputAdornment>
+          ),
+        }}
+        helperText="Hint: use combination of two-three words, mix uppercase with lowercase letters, and use special characters"
```
Comment (nl0) [inline catalog/app/containers/Auth/SignUp.js:63]: shouldnt we use hints from `zxcvbn`?

```
result.feedback   # verbal feedback to help choose better passwords. set when score <= 2.

  result.feedback.warning     # explains what's wrong, eg. 'this is a top-10 common password'.
                              # not always set -- sometimes an empty string

  result.feedback.suggestions # a possibly-empty list of suggestions to help choose a less
                              # guessable password. eg. 'Add another word or two'
```

**Review by nl0 — APPROVED** [review]
looks really nice and useful, tho i'd try showing `zxcvbn`'s feedback since it seems quite relevant and based on the actual user input (not some generic advice).
also, we should implement the same constraints on the backend probably

File: catalog/app/containers/Auth/SignUp.js (hunk 1 above)
Comment (nl0) [reply catalog/app/containers/Auth/SignUp.js]: > I use result.feedback.suggestions with the new changeset. But warning almost always is empty for me and shows some strange warnings

ok, you can revert if you think it's not useful (i trust your judgement)

Comment (fiskus) [reply catalog/app/containers/Auth/SignUp.js]: Suggestions are useful, warning for some reason - not useful :man_shrugging:

**Pull request merged**